src/reducers/authReducer.js

```diff
@@ -34,6 +34,21 @@ const authReducer = (state = intialState, action) => {
                 }
             }
 
+            if (decodedAccessToken.role === userRoles.LOGIST) {
```

Comment:
It is necessary to make from 2 if — one in which condition will be checked whether the role is in an array of userRoles. And in the body — tokens are set and change the role & the isAuthUser from the state